ckan/controllers/package.py

```diff
@@ -165,7 +165,8 @@ def remove_field(key, value=None, replace=None):
 
         c.remove_field = remove_field
 
-        sort_by = request.params.get('sort', None)
+        sort_by = request.params.get('sort', 'metadata_modified desc')
+        request.GET['sort'] = sort_by
```

Comment:
It is to pass the default sort to template so that template may not have to rely on sort==None to decide the default selected option.